pkgs/by-name/hy/hyprwall/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  fetchurl,
+  fetchFromGitHub,
+  rustPlatform,
+
+  pkg-config,
+  glib,
+  gdk-pixbuf,
+  pango,
+  graphene,
+  gtk4,
+  gtk3,
+  git,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hyprwall";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "hyprutils";
+    repo = "hyprwall";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-dzPd+5cws3hKhdd1CKKEO7EWMS0XW0y1vqxg1XKX+gY=";
+  };
+
+  cargoHash = "sha256-gT2ysWHckcUl1yx5tciy6kSvZZ0srrs4OwI1mr/58Pc=";
+
+  nativeBuildInputs = [
+    pkg-config
+    glib
+    gdk-pixbuf
+    pango
+    graphene
+    gtk4
+    gtk3
+    git
+  ];
+
+  buildInputs = [
+    pkg-config
+    glib
+    gdk-pixbuf
+    pango
+    graphene
+    gtk4
+    gtk3
+  ];
```

Comment:
```suggestion
  nativeBuildInputs = [
    pkg-config
    glib
    pango
    wrapGAppsHook4
  ];
```

Reply:
Not sure what is `git` used for?

Reply:
Git was unnecessary, build fails without the gtk3 dependency as a buildInput so I've kept it.

Reply:
Could you leave a comment there?